atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/KvsGetDynamicBenchmarks.java

```diff
@@ -50,31 +50,31 @@
     @Benchmark
     @Warmup(time = 20, timeUnit = TimeUnit.SECONDS)
     @Measurement(time = 180, timeUnit = TimeUnit.SECONDS)
-    public Object getAllColumnsExplicitly(WideRowTable table) {
+    public Object getAllColumnsExplicitly(ModeratelyWideRowTable table) {
         Map<Cell, Value> result = table.getKvs().get(table.getTableRef(), table.getAllCellsAtMaxTimestamp());
-        Preconditions.checkState(result.size() == WideRowTable.NUM_COLS,
-                "Should be %s columns, but were: %s", WideRowTable.NUM_COLS, result.size());
+        Preconditions.checkState(result.size() == table.getNumCols(),
+                "Should be %s columns, but were: %s", table.getNumCols(), result.size());
```

Comment:
Yeah, it seems we're building the same string everywhere. I'm fine either way, in any case.